exersice1.js

```diff
@@ -0,0 +1,100 @@
+// You are talking to a friend who suddently looks at you 
+// and asks: "How old will you be in 2045?" Hmm you remember 
+// the year you were born and try to do some calculation:
+// Born in 1987 + 3 that's 1990. 90 - 40 is 50 + 5 thats 58. 
+// I will be 58! Wow that was painful! Let's fix that by making 
+// some code that automatically does this!
+
+let yearOfBirth = 1986;
+let yearFuture = 2051;
+let age;
+
+age = yearFuture - yearOfBirth;
+
+console.log(`You will be ${age} old in ${yearFuture}`);
+
+// Goodboy-Oldboy (A dog age calculator)
+// The same friend(who by the way loves dogs) asks how old his 
+// dog will be in 2045. Hmm you think, lets make this into a
+// product as well! Dogs age can either be measured in dog years
+// or in human years, this we want to take into consideration!
+
+const dogYearOfBirth = 2022;
+const dogYearFuture = 2025;
+let dogYear;
+
+dogYear = dogYearFuture - dogYearOfBirth;
+
+const shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears === false) {
+  console.log(`Your dog will be ${dogYear * 7} dog years old in ${dogYear}`);
+} else {
+  console.log(`Your dog will be ${dogYear} human years old in ${dogYear}`);
```

Comment:
I think there should be `dogYearFuture` instead of `dogYear` in the console log statements